pkg/controller/controller_utils.go

```diff
@@ -952,7 +952,7 @@ func FilterActivePods(logger klog.Logger, pods []*v1.Pod) []*v1.Pod {
 		if IsPodActive(p) {
 			result = append(result, p)
 		} else {
-			logger.V(4).Info("Ignoring inactive pod", "pod", klog.KObj(p), "phase", p.Status.Phase, "deletionTime", p.DeletionTimestamp)
```

Comment:
leave this for a separate PR

Reply:
Just wanted to make sure the constant panics don't impact the benchmarks.

Reply:
Done